docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+   can set up pull requests to our main code package (i.e, suggest changes) if you find
+   errors and have the packages/dependencies automatically installed.
+
+Why do you recommend installing and submitting the way outlined in this guide? In other words, why should I do it your way?
+------------------------------------------------------------------------------
+
+A reasonable question, and it is always good to be skeptical. The short answer
+is that using an IDE like Pycharm and virtual environments
+drastically cuts the error rate for install down, as well as makes the whole
+process of installing dependencies easier. Using a venv also helps with headaches
+caused by clashes between Anaconda and PIP, and Pycharm
+(or another IDE like VScode) takes care of that for you.
+
+Do I have to read/do this entire tutorial to submit a model?
+------------------------------------------------------------
+
+No - You can skip to the Quickstart section below, if you do not
+wish to read/do this entire tutorial, but we recommend it in
+order to get the most out of Brain-Score and avoid errors that might pop up along the way.
+
+
+
+Quickstart
+==========
+In this section, we will provide a quick and easy way to get Brain-Score
+up and running. This is mainly for those who do not have the time to read
+or do the whole tutorial, or for those who just want to go ahead and submit
+a model quickly; however, we do officially recommend that you read and do the
+entire tutorial, as it will help you in the future. This section also does not
+have pictures, which the other more lengthy sections below do. In this case,
+we will submit a version of AlexNet from Pytorch’s library; the main steps are outlined below:
+
+1. Clone a copy of  the ``sample-model-submission`` repo from our Github. If you are wondering
+   about what the various repos on the main Brain-Score github do, check out the ``Overview`` section above.
+   Here is the command for a terminal: ::
+     git clone https://github.com/brain-score/sample-model-submission.git
+2. Install the dependencies via pip. (Make sure to switch into the ``sample-model-submission`` folder
+   that was created in step one above when you cloned the repo). You can read more about this in the
+   ``Install Brain-Score Repos and Dependencies`` section below. Again, the command for the terminal is: ::
+     pip install .
+3. Specify the model to test. You can do this a few ways, but in this case, we can
+   just use the model defined in ``pytorch.py``. More info on this step can be found in
+   the ``Submitting a Model to Brain-Score.org Part 1: Preparing the Model`` section of
+   this guide. In essence, you need to implement the various functions outlined in
+   the ``sample-model-submission/models/base_models.py`` file.
+4. Test the model on your machine. You can do this simply by hitting “run"
+   on the ``pytorch.py`` file (in this case) or the file where you implemented
```

Comment:
"by hitting “run" on the ``pytorch.py`` file" -> "by running the ``pytorch.py`` file"

as above, I'm not sure what the `pytorch.py` file is